### PR TITLE
Clockwork Helmets will now protect from the 'bang' portion of flashbangs

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Clockwork Helmets will now protect from the 'bang' portion of flashbangs.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
unnecessary

**Comment by ChangelingRain** [comment]
I'm going to buff clockcult until people stop saying it's weak

**Comment by PKPenguin321** [comment]
up next: clock cult gets a button that calls del-all on tasers/stunbatons

**Comment by ChangelingRain** [comment]
they already have that it's called vanguard

**Comment by TechnoAlchemisto** [comment]
Clockcult strength is bottlenecked at their conversion rate, their kit is already extremely strong once they have even a small group of converted people congregated.

**Comment by Xhuis** [comment]
I agree with PK in that this isn't really necessary. Clockcult has so many different convoluted tools and structures now that it seems - to me, at least - that there's a huge amount of situationally useful stuff but nothing that's actually generally good. Having a tool for every situation isn't necessarily bad, but someone who knows how to use them is basically God compared to a normal crewman, and they shouldn't have a counter to _everything_ built in (like this buff).

tl;dr Buffs bad, reworks good
